### PR TITLE
Papertrail update

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: config.yaml.defaults [hunk 1]
```diff
@@ -62,17 +62,15 @@ ports:
...
-           port: XX
-           # which log files, if any do you not want to send over to the 3rd party?
-           excluded_log_files: []
+    papertrail:
+       # get an account at https://papertrailapp.com
+       enabled: False
+       # change url to the correct subdomain for your account
+       # and change the port as well
+       url: logX.papertrailapp.com
+       port: XX
```
Comment (profjsb) [inline config.yaml.defaults:71]:
```suggestion
       port:
```

File: config.yaml.defaults [hunk 2]
```diff
@@ -62,17 +62,15 @@ ports:
...
-           url: logX.papertrailapp.com
-           port: XX
-           # which log files, if any do you not want to send over to the 3rd party?
-           excluded_log_files: []
+    papertrail:
+       # get an account at https://papertrailapp.com
+       enabled: False
+       # change url to the correct subdomain for your account
+       # and change the port as well
+       url: logX.papertrailapp.com
```
Comment (profjsb) [inline config.yaml.defaults:70]:
```suggestion
       url:
```

**Review by profjsb — APPROVED** [review]
Looks good!

File: tools/watch_logs.py [hunk 3]
```diff
@@ -45,7 +45,7 @@ def logs_from_config(supervisor_conf):
 watched.append('log/nginx-bad-access.log')
 watched.append('log/nginx-error.log')
 watched.append('log/fake_oauth2.log')
-
+watched.append('log/app_00.log')
```
Comment (profjsb) [inline tools/watch_logs.py:48]: Do you want to watch other files too?

Comment (stefanv) [reply tools/watch_logs.py:48]: I think there's a bigger problem we need to address here, which is that we should pick up on new logfiles as they appear and add threads for them.  I'll fix that in another PR.

**Pull request merged**